apps/meteor/client/views/account/preferences/PreferencesNotificationsSection.tsx

```diff
@@ -153,6 +171,18 @@ const PreferencesNotificationsSection = ({ onChange, commitRef, ...props }: Form
 						{!canChangeEmailNotification && t('Email_Notifications_Change_Disabled')}
 					</Field.Hint>
 				</Field>
+
+				{showNewLoginEmailPreference && (
+					<Field>
+						<Box display='flex' flexDirection='row' justifyContent='spaceBetween' flexGrow={1}>
```

Comment:
I'm surprised by how many `justifyContent='spaceBetween'` references exist in the Preferences' components... It should be `justifyContent='space-between'`, as only the prop name follows the naming convention of properties from `CSSProperties`.